htdocs/societe/list.php

```diff
@@ -390,23 +404,20 @@
 	}
 } else dol_print_error($db);
 
-$sql = "SELECT s.rowid, s.nom as name, s.name_alias, s.barcode, s.address, s.town, s.zip, s.datec, s.code_client, s.code_fournisseur, s.logo,";
+$sql  = "SELECT DISTINCT s.rowid, s.nom as name, s.name_alias, s.barcode, s.address, s.town, s.zip, s.datec, s.code_client, s.code_fournisseur, s.logo,";
```

Comment:
The distinct should not be required.
If we search on 1 category, we can have a left join on the id of category, so we will have no duplicated record.
And if we need a AND, the left join is no more required, only a AND added into where will be enough (a subrequest to test a condition on id of company into categorie_societe)

Reply:
You're right, it's no longer appears to be necessary.

Reply:
Sorry, it's useful for OR operator in categories filters